app/src/main/java/com/artemzin/qualitymatters/ApplicationModule.java

```diff
@@ -33,6 +34,11 @@ public QualityMattersApp provideQualityMattersApp() {
     }
 
     @Provides @NonNull @Singleton
+    public Context provideContext() {
```

Comment:
Ok, I'll change the method signature in the @Module to provide the app to the constructor. Just curious, why exactly are you afraid of ```Context```?